ckanext/opendata_theme/opengov_custom_theme/templates/base.html

```diff
@@ -0,0 +1,14 @@
+
+{% block head_extras %}
+  {{ super() }}
+  <script>
+    var enabled = "{{ h.opendata_theme_is_segment_active() | escapejs }}";
+    var apiKey = "{{ h.opendata_theme_segment_apikey() | escapejs }}";
+    if(enabled === "True" && apiKey !== ""){
```

Comment:
If `opendata_theme_is_segment_active()` is updated to return a boolean this should be updated as well

Reply:
Also the unit tests are failing as there is no `escapejs` filter in Jinja